File: WEEK1/DAY4/Prog1/P1.c
#include <stdio.h>
int main(){
    int N;
    printf("Print multiplication table for N\n");
    printf("Enter N: ");
    scanf("%d",&N);
    for(int i=1;i<=12;i++){
        printf("%d * %d = %d\n",N,i,N*i);
    }
}

File: WEEK1/DAY4/Prog1/P1.java
import java.util.*;
public class P1 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Print multiplication table for N");
        System.out.print("Enter N: ");
        int N=sc.nextInt();
        for(int i=1;i<=12;i++){
            System.out.println(N+"*"+i+"="+(N*i));
        }
        sc.close();
    }
}